bin/disco_deploy.py

```diff
@@ -28,6 +29,7 @@
      --hostclass HOSTCLASS  Limit command to a specific hostclass
      --environment ENV      Environment to operate in
      --allow-any-hostclass  Do not limit command to hostclasses defined in pipeline
+     --strategy STRATEGY    The deployment strategy to use. Currently supported: 'classic' or 'blue_green'.
```

Comment:
You can't really state a default here because that'll break things because docopt is too cute for its own good.

The default is documented in the README though. Good enough?